.changeset/healthy-penguins-return.md

```diff
@@ -0,0 +1,11 @@
+---
+'example-app': patch
```

Comment:
You don't have to bump the `example-app`

Reply:
I'm adding a file to it. Isn't that a patch-level version bump? It's not the same exact version any more.

Reply:
The example app is not a published package, it's only for local development, so there is no need to change the version.